SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,72 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     |                    2+ |
+    |                  |                                         | - FSS        |                       |
+    |                  |                                         | - SPDZ       |                       |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Subtraction                             | - Falcon     |                    2+ |
+    |                  |                                         | - FSS        |                       |
+    |                  |                                         | - SPDZ       |                       |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Mult, matmul, conv2d, conv2d _transpose | - Falcon     | - 3                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Equal to                                | - FSS        |                    2+ |
```

Comment:
I think FSS is a 2 PC Framework, the paper mentions extension for n-party FL scenario,we support only 2 PC scenario for the ops @LaRiffle @gmuraru

Reply:
Yep - it is 2 parties only -- it can be extended, but the protocol supports (officially) only 2 parties